css/properties/mask-border-mode.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I'd drop this file. If it's unimplemented and there's no sign of implementation, then I don't think it should be in BCD.

(I know I need to resolve the whole content linter problem with missing BCD. Actually, there's a few problems with it now—thank you for your patience while I get to it.)